### DISCUSSION TITLE
Change font

### DISCUSSION THREAD

**Discussion by javoxmg** [fmarotta/kaobook#224] [Q&A]
Hi Federico,

First of all, thank you so much for this project. I think is a really cool latex class. I am actually thinking about using it for a book I am writing.

The only problem I have so far is that I am not totally convinced by the font, specially for the math. I spent an hour diving into the files and I see references everywhere to \normalfont but no way to change such a font. Would it be possible to change it back to the more standard Latin Modern font?

Thank you very much and kind regards :)

**Comment by aryacliche** [comment]
Edit : This method (i.e. step 1 from the end of the comment) worked. Apparently it needed me to restart my laptop to show the change?
*****

I also spent the last two hours trying to get the math font changed to Computer Modern or Latin Modern Math. I am very much of a beginner with LaTeX so my comment may not help much.

I can see the following lines in `kao.sty` which seem to decide the math font (and based on documentation, seem to govern the math font among others),
```
\ifxetexorluatex
	\RequirePackage{amssymb} % Must be loaded before unicode-math
	\RequirePackage[force]{filehook} % Fixes an error
	\RequirePackage{unicode-math} % Math fonts in xetexorluatex
    \setromanfont[
		Scale=1.04
	]{Libertinus Serif}
	\setsansfont[
		Scale=1
	]{Libertinus Sans}
	\setmonofont[
		Scale=.89
	]{Liberation Mono}
	\setmathfont{Libertinus Math} % [CUSTOM COMMENT 1]
    \ifluatex
    \else
        \RequirePackage{morewrites} % Fix some errors related to floats
    \fi
\else
	\RequirePackage[utf8]{inputenc} % utf8 encoding in the input (.tex) file
	\RequirePackage[T1]{fontenc} % utf8 encoding in the output (.pdf) file

	\RequirePackage{amssymb} % Math symbols, including \blacktriangleright, used for bullets
	\RequirePackage[scaled=.97,helvratio=.93,p,theoremfont]{newpxtext} % Serif palatino font
	\RequirePackage[vvarbb,smallerops,bigdelims]{newpxmath} % Math palatino font % [CUSTOM COMMENT 2]
	\RequirePackage[scaled=.85]{beramono} % Monospace font
	\RequirePackage[scr=rsfso,cal=boondoxo]{mathalfa} % Mathcal from STIX, unslanted a bit % [CUSTOM COMMENT 3]
    \RequirePackage{morewrites} % Fix some errors related to floats
\fi
```
The lines tagged as `[CUSTOM COMMENT x]` deal with math fonts. For context, I am using **PDFLatex** with **latexmk** to compile my tex files. I have tried to also delete all temporary (or auxillary) files before each compilation to get a clean build.

Here are the things I tried,
1. I commented out the line tagged `[CUSTOM COMMENT 2]` and `[CUSTOM COMMENT 3]` as well as replacing them with `\RequirePackage[]{lmodern}`. 
2. I even replaced `Libertinus Math` with `Latin Modern Math` at `[CUSTOM COMMENT 1]` even though it doesn't apply to my engine.
3. I tried overriding this by using `DeclareMathVersion` and `setmathfont` in my `main.tex` files. I am getting errors in both cases.

None of the above have resulted in any difference in the math font rendered. I am also clueless beyond this.
